firmware/src/simple_model.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * simple_model.h
+ * Copyright (C) 2015 Simon Newton
+ */
+
+/**
+ * @defgroup simple_model Simple RDM Model
+ * @brief A simple RDM Model for controlling RGB pixels.
```

Comment:
The simple one isn't doing RGB pixels (yet). Is it worth the simple one just being a straight dummy one anyway.